web/themes/new_weather_theme/templates/partials/satellite.html.twig

```diff
@@ -1,26 +1,24 @@
 {% if weather.satellite %}
-<div class="grid-container margin-top-2 padding-x-0 tablet:padding-x-2" wx-outer-radar-container>
-  <div class="grid-row tablet-lg:grid-col-8 tablet-lg:grid-offset-2">
-    <h3 class="wx-visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">{{ "Satellite" | t }}</h3>
-  </div>
+  <div class="margin-top-2" wx-outer-radar-container>
+    <h3 class="wx-visual-h2 text-normal text-primary-darker margin-top-3 margin-bottom-2">{{ "Satellite" | t }}</h3>
```

Comment:
After removing the padding on mobile, needs to add `padding-x-2 tablet:padding-x-0` so that this heading lines up with the others.